linuxkit_template.yml

```diff
@@ -0,0 +1,61 @@
+kernel:
+  image: linuxkit/kernel:4.9.75
```

Comment:
I would prefer this not to be in the top-level. Maybe `./examples` would be a better place. Also having a test which `bake`s the template and then builds it would be useful.

Reply:
done, lets see what ci says